crossdock/server/server.go

```diff
@@ -77,6 +85,9 @@ func (s *Server) Start() error {
 			return s.doJoinTrace(ctx, req.(*tracetest.JoinTraceRequest))
 		})
 	})
+	// TODO test_driver host needs to be initialized before this
+	//mux.HandleFunc("/endtoend", s.eHandler.Trace)
```

Comment:
* so you're *not* adding a new endpoint?
* I suggest reconsidering end-to-end name. The relevant endpoint name seems to be "create_spans".